library/ssl_tls.c

```diff
@@ -1852,27 +1852,55 @@ void mbedtls_ssl_set_verify( mbedtls_ssl_context *ssl,
 #endif
 
 #if defined(MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED)
-/*
- * Set EC J-PAKE password for current handshake
- */
+
 #if defined(MBEDTLS_USE_PSA_CRYPTO)
+static psa_status_t mbedtls_ssl_set_hs_ecjpake_password_common(
```

Comment:
Note (no change required): static functions don't need the `mbedtls_` prefix so we can skip it it we want in order to make names shorter and distinguish from public function. But it's also absolutely OK to keep it.